### PR TITLE
Modify add tags placeholder

### PR DESCRIPTION
**Pull request by uzmannazari**
## Summary
Modify add tags place holder in add and edit pages that can be read from language resource files

### CODE REVIEW

**Review by mitchelsellers — CHANGES_REQUESTED** [review]
This needs to have an edit to the localization file to include this new element.

File: Dnn.AdminExperience/ClientSide/Dnn.React.Common/src/Tags/index.jsx [hunk 1]
```diff
@@ -185,7 +185,7 @@ Tags.propTypes = {
 };
 
 Tags.defaultProps = {
-    addTagsPlaceholder: "Add Tags",
+    addTagsPlaceholder: "{Localization.get("addTagsPlaceholder")}",
     searchTagsPlaceholder: "Begin typing to search tags",
```
Comment (valadas) [inline Dnn.AdminExperience/ClientSide/Dnn.React.Common/src/Tags/index.jsx:189]: Also, could be nice to do this other one in the same go...

**Review by valadas — CHANGES_REQUESTED** [review]

**Comment by uzmannazari** [comment]
> This needs to have an edit to the localization file to include this new element.

Localization file has this resource but its not be applied on this file

File: Dnn.AdminExperience/ClientSide/Dnn.React.Common/src/Tags/index.jsx (hunk 1 above)
Comment (uzmannazari) [reply Dnn.AdminExperience/ClientSide/Dnn.React.Common/src/Tags/index.jsx:189]: you mean this element on line 188? **searchTagsPlaceholder**

Comment (valadas) [reply Dnn.AdminExperience/ClientSide/Dnn.React.Common/src/Tags/index.jsx:189]: Yeah, also, should'nt the double-quotes be removed in this context ?

Comment (uzmannazari) [reply Dnn.AdminExperience/ClientSide/Dnn.React.Common/src/Tags/index.jsx:189]: Yes, I also agree

Comment (uzmannazari) [reply Dnn.AdminExperience/ClientSide/Dnn.React.Common/src/Tags/index.jsx]: @valadas but i need to edit resources now and i cant find the right file address in the source and i don't know how to edit more files for 1 pull request!

**Review by valadas — APPROVED** [review]
Awesome, thanks for the addition

**Comment by uzmannazari** [comment]
> Awesome, thanks for the addition

I have come across a problem in testing the changes on the DNN source. can you please help me?

**Review by valadas — CHANGES_REQUESTED** [review]
I believe the double-quotes need to be removed here

**Comment by valadas** [comment]
> > Awesome, thanks for the addition
> 
> I have come across a problem in testing the changes on the DNN source. can you please help me?

You can run ./build.ps1 then install a site from the packages in the artifacts folder. If that gives you problems, please describe and we'll try to help

File: Dnn.AdminExperience/ClientSide/Dnn.React.Common/src/Tags/index.jsx [hunk 2]
```diff
@@ -185,8 +185,8 @@ Tags.propTypes = {
 };
 
 Tags.defaultProps = {
-    addTagsPlaceholder: "Add Tags",
-    searchTagsPlaceholder: "Begin typing to search tags",
+    addTagsPlaceholder: "{Localization.get("addTagsPlaceholder")}",
```
Comment (mitchelsellers) [inline Dnn.AdminExperience/ClientSide/Dnn.React.Common/src/Tags/index.jsx:188]:
```suggestion
    addTagsPlaceholder: {Localization.get("addTagsPlaceholder")},
```

**Review by mitchelsellers — APPROVED** [review]

**Comment by valadas** [comment]
@uzmannazari we are now doing some open-source co-coding session each Saturday 7PM (EST) on Discord if you want to join use, we are happy to help you get this PR across the line... https://discord.gg/m4RaJWd8

**Review by valadas — APPROVED** [review]

File: Dnn.AdminExperience/ClientSide/Dnn.React.Common/src/Tags/index.jsx [hunk 3]
```diff
@@ -185,8 +185,8 @@ Tags.propTypes = {
 };
 
 Tags.defaultProps = {
-    addTagsPlaceholder: "Add Tags",
-    searchTagsPlaceholder: "Begin typing to search tags",
+    addTagsPlaceholder: {Localization.get("addTagsPlaceholder")},
+    searchTagsPlaceholder: {Localization.get("searchTagsPlaceholder")},
```
Comment (valadas) [inline Dnn.AdminExperience/ClientSide/Dnn.React.Common/src/Tags/index.jsx:189]: I am not sure code can run in defaultProps definition, I am not 100% sure but the build is failing

**Review by valadas — CHANGES_REQUESTED** [review]
This needs to be fixed AND TESTED to be merged. If you need help getting this working/tested feel free to join us on Discord and we can help: https://discord.gg/UBXKjneT5Y

**Comment by valadas** [comment]
yeah, we don't have Localization methods in the common components, the consuming code needs to override this prop where it is used. Can't merge this PR in this state, sorry.

**Pull request closed without merging**